packages/launchpad/src/setup/EnvironmentSetup.cy.tsx

```diff
@@ -15,7 +15,7 @@ describe('<EnvironmentSetup />', { viewportWidth: 800 }, () => {
     })
 
     cy.findByRole('button', {
-      name: 'Front-end Framework Pick a framework',
+      name: 'Front-end framework Pick a framework',
```

Comment:
This is a strange use of the `name` attribute on a button. No changes requested here, just making a note that we should look into it